values.yaml

```diff
@@ -83,7 +83,16 @@ kpi:
       # AWS_SECRET_ACCESS_KEY: ""
   worker:
     replicaCount: 1
-    resources:
+    livenessProbe:
+      enabled: true
+      initialDelaySeconds: 10
+      periodSeconds: 60
+      timeoutSeconds: 30
+      command:
+        - /bin/sh
+        - -c
+        - exec
```

Comment:
To make it work, I had to remove `exec` from the command. 
With  `exec`, event `Unhealthy` seems to never be triggered.  

To test, I just stopped redis and wait for the pod to restart.
I decreased `periodSeconds` to 30 and `timeoutSeconds` to 10 to test. Pod should restart within the next two minutes Redis has been stopped.

Reply:
The only variation that I could get working was this

```
    command:
      - "bash"
      - "-c"
      - "celery -A glitchtip inspect ping -d celery@$HOSTNAME | grep -q OK"
```

I will go ahead and switch it to `{{- .Values.kpi.worker.livenessProbe | toYaml | nindent 12 }}` as @Yann-J suggested to cover those additional options. And make the change to the upstream django-helm-chart.

I tested by starting and stopping redis. I'm confused why the /bin/sh version doesn't appear to work for me.

Reply:
> `"celery -A glitchtip` 

Maybe workers are not using the same base image in GT and Kobo (I tested it with Kobo).

Reply:
Let me double check - I would explicitly like to keep aligned to django-helm-chart which we use and glitchtip uses.

Reply:
Wow you're right - I can't explain it though. Both projects use python:3.10-slim as the base image. I still think we should keep aligned to django-helm-chart and that it needs to use bash as it works in more observed cases. If this looks good as is, please merge.